### PR TITLE
Add notes on -p usage

### PR DESCRIPTION
**Pull request by josegonzalez**

### CODE REVIEW

**Comment by progrium** [comment]
I don't believe builtins are disabled. But preprocessing with posix interpolation *can* break your template. For example `{{ $foo }}` would pre-render to ` {{ }}` if $foo is empty. Definitely the behavior of `-p` has changed since the docs were written.

**Comment by josegonzalez** [comment]
The docs changes are based on real-world usage by @michaelshobbs, so maybe he can chime in.

**Comment by michaelshobbs** [comment]
Example below
```
$ cat test.tmpl
{{if .VAR1}}
here is a POSIX style variable: $VAR1
{{end}}
{{if .VAR1}}
here is a golang style variable: {{.VAR1}}
{{end}}

{{include "test.include"}}
```
```
$ cat test.include
Hello World from an include!!
```
```
$ sigil -f ./test.tmpl VAR1="Hello World"

here is a POSIX style variable: $VAR1


here is a golang style variable: Hello World


Hello World from an include!!

root@dokku:~/dokku/plugins/nginx-vhosts/templates# sigil -p -f ./test.tmpl VAR1="Hello World"
template: test.tmpl:8: function "test" not defined
root@dokku:~/dokku/plugins/nginx-vhosts/templates#
```

**Comment by lalyos** [comment]
@michaelshobbs with version 0.4.0 it works fine.

File: README.md [hunk 1]
```diff
@@ -28,11 +28,16 @@ Other releases can be downloaded from [Github Releases](https://github.com/glide
...
-used as variables.
+used as variables. You can use the `-p` flag to enable POSIX variable expansion, 
+like so:
 
  * `echo 'Hello, $name' | sigil -p name=Jeff`
  * `sigil -p -f config.tmpl var1=foo "var2=Hello world"`
 
+Without this flag, you will need to use double-brackets for variables:
+
+ * `echo 'Hello, {{name}}' | sigil name=Jeff`
```
Comment (lalyos) [inline README.md]: You have to use either  `{{$name}}` or `{{.name}}`

**Comment by michaelshobbs** [comment]
@lalyos version 0.4.0? I don't see that on the releases page. Where did you find it?

**Comment by lalyos** [comment]
@michaelshobbs right its not cut out yet. So just open #15 to perform it.
Meanwhile i did it on my fork, so you can try:
```
curl -L https://github.com/lalyos/sigil/releases/download/v0.4.0/sigil_0.4.0_$(uname)_x86_64.tgz | tar -xz -C /usr/local/bin/
```

**Pull request closed without merging**